doc/source/train/user_guide.rst

```diff
@@ -328,168 +328,184 @@ perform hyperparameter tuning with Ray Train, please refer to the
 
 .. _train-result-object:
 
-The Result Object
------------------
+Accessing Training Results
+--------------------------
 
 .. TODO(ml-team) Flesh this section out.
 
 The return of a ``Trainer.fit`` is a :class:`Result` object, containing
 information about the training run. You can access it to obtain saved checkpoints,
 metrics and other relevant data.
 
-.. _train-log-dir:
+.. automodule:: ray.air.result
+    :members:
+    :noindex:
```

Comment:
Seems a bit weird to include this directly in the user guide - I think it's okay to remove this as long as the previous reference links to the API docs (which right now isn't happening).